apps/src/applab/ConnectedStudioAppWrapper.jsx

```diff
@@ -0,0 +1,19 @@
+/** @file Redux-connected version of StudioAppWrapper, used by App Lab. */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var connect = require('react-redux').connect;
+var StudioAppWrapper = require('../templates/StudioAppWrapper.jsx');
+
+module.exports = connect(
+  (state) => ({
+    assetUrl: state.assetUrl,
+    isEmbedView: state.isEmbedView,
+    isShareView: state.isShareView
+  })
+)(StudioAppWrapper);
```

Comment:
I suppose not, given that it's got a pretty clearly distinct name. I guess I'm fine with this, as I don't really have much justification for my original distaste.